app/javascript/packages/document-capture/components/file-input.jsx

```diff
@@ -202,7 +169,7 @@ const FileInput = forwardRef((props, ref) => {
         onDrop={() => setIsDraggingOver(false)}
       >
         <div className="usa-file-input__target">
-          {value && !isMobile && (
+          {value && value instanceof window.File && !isMobile && (
```

Comment:
The differences between `File` and `Blob` interfaces give us a nice way to differentiate between named and unnamed blobs of image data, where an uploaded file will have a name, but an image captured from Acuant wouldn't have a name (unless we generated one). So in preview, we can present the name of the selected file based on whether the value is `instanceof File`.

- https://developer.mozilla.org/en-US/docs/Web/API/Blob
- https://developer.mozilla.org/en-US/docs/Web/API/File (`File extends Blob`)

Reply:
> should we wrap up that logic in functions? `isFileWithName()` and `isFileWithoutName()` maybe?

I personally feel it's already well represented by the native distinction between `Blob` and `File` types, where `File` includes additional properties (one of which being `name`). I guess I do this a disservice by naming and documenting the `Blob` types as "files", though that was more a pragmatic choice, since it's fair to say it's "file-like", and that the term "blob" isn't as readily understandable.

Reply:
Ok! I guess for me `instanceof` checks are always a little suspicious since I'm used to more duck-typing-like approaches in Ruby